shared/src/Datadog.Trace.ClrProfiler.Native/single_step_guard_rails.cpp

```diff
@@ -81,14 +81,14 @@ HRESULT SingleStepGuardRails::CheckRuntime(const RuntimeInformation& runtimeInfo
             }
 
             // .NET Core 3.1+, but is it _too_ high?
-            if(runtimeInformation.major_version <= 9)
+            if(runtimeInformation.major_version <= 10)
```

Comment:
Should we make note of this that SSI will now automatically support .NET 10?

Reply:
I think putting in the release notes is good enough, but we can ping internal teams too. Were you thinking about anything else?